### PR TITLE
Adding more sysmetrics to Oracle monitoring 

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do?

We are adding more sysmetrics to Oracle monitoring. 

### Motivation

The goal is to improve the observability from the DBA/SRE/system engineer perspective .

### Additional Notes

The metrics will be displayed in the new Oracle dashboard: https://dd.datad0g.com/dashboard/u7k-cy2-b9d/oracle?from_ts=1685883639120&to_ts=1685970039120&live=true

### Describe how to test/QA your changes

Start the agent and verify that the metrics are displayed on the dashboard.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by jmeunier28 — APPROVED** [review]

File: pkg/collector/corechecks/oracle-dbm/sysmetrics.go [hunk 1]
```diff
@@ -9,59 +9,151 @@ import (
...
 }
 
+type sysMetricsDefinition struct {
+	DDmetric string
+	DBM      bool
+}
+
+var SYSMETRICS_COLS = map[string]sysMetricsDefinition{
+	"Average Active Sessions":                       {DDmetric: "active_sessions"},
+	"Average Synchronous Single-Block Read Latency": {DDmetric: "avg_synchronous_single_block_read_latency", DBM: true},
```
[the remark this answers is not in the document]
Comment (nenadnoveljic) [reply pkg/collector/corechecks/oracle-dbm/sysmetrics.go:44]: It's a decision we have to make: how much do we want to give away for free? All new metrics that hadn't been emitted so far, I put under the DBM license in the first version, and I'm agnostic about it.

**Review by cswatt — APPROVED** [review]
release notes look good

File: pkg/collector/corechecks/oracle-dbm/sysmetrics.go [hunk 2]
```diff
@@ -9,59 +9,151 @@ import (
...
+	"User Rollbacks Per Sec":                        {DDmetric: "user_rollbacks"},
+}
+
+func (c *Check) sendMetric(s aggregator.Sender, r SysmetricsRowDB, seen map[string]bool) {
+	if metric, ok := SYSMETRICS_COLS[r.MetricName]; ok {
+		value := r.Value
+		if r.MetricUnit == "CentiSeconds Per Second" {
+			value = value / 100
+		}
+		if !SYSMETRICS_COLS[r.MetricName].DBM || SYSMETRICS_COLS[r.MetricName].DBM && c.dbmEnabled {
```
Comment (alexandre-normand) [inline pkg/collector/corechecks/oracle-dbm/sysmetrics.go:118]: This doesn't have to change now but if DBM is disabled and some metrics are not going to get sent, should we skip the collection of the values entirely?

Comment (nenadnoveljic) [reply pkg/collector/corechecks/oracle-dbm/sysmetrics.go:118]: A `WHERE` condition wouldn't be selective. I haven't measured it, but a full table scan might be cheaper than a filter with many `OR` operands. We can do a benchmark at some less busy times.

Comment (alexandre-normand) [reply pkg/collector/corechecks/oracle-dbm/sysmetrics.go:118]: Oh right. This makes total sense to do the filtering in post-processing then.

**Review by alexandre-normand — APPROVED** [review]

**Pull request merged**